docs/src/content/en/docs/workflows-vnext/inngest-workflow.mdx

```diff
@@ -7,59 +7,84 @@ description: "Inngest workflow allows you to run Mastra vNext workflows with Inn
 
 [Inngest](https://www.inngest.com/docs) is a developer platform for building and running background workflows, without managing infrastructure.
 
-## Setup
+## How Inngest Works with Mastra
 
-```sh
-npm install @mastra/inngest @mastra/core @mastra/deployer @hono/node-server
-```
+Inngest provides an event delivery and orchestration layer for running Mastra vNext workflows in response to events. The integration connects Inngest’s event system to Mastra’s workflow engine, enabling event-driven execution, step-level monitoring, and suspend/resume functionality.
 
-### Local Development Environment
+### Integration Architecture
 
-Inngest offers two methods for local development:
+- **Workflow Registration:**  
+  Workflows are defined using the `createWorkflow` and `createStep` helpers from `@mastra/inngest`. These helpers produce workflow and step objects compatible with both Mastra and Inngest.
```

Comment:
We probably should describe how these are registered into inngest `serve` function as well.